app/routes/events/components/EventEditor/index.js

```diff
@@ -79,6 +81,13 @@ function EventEditor({
   initialized,
 }: Props) {
   const isEditPage = eventId !== undefined;
+
+  const [useMazemap, setUseMazemap] = useState<boolean>(false);
+
+  useEffect(() => {
+    setUseMazemap(event.mazemapPoi && event.mazemapPoi?.value > 0);
+  }, [initialized]);
+
```

Comment:
You could have done this like we have with the `separateDeadlines` for consistency, but seeing as this is the more "modern" method and we avoid some redux I think this probably is better :+1:

Reply:
:)